iesetup, iewzd, iexplore, iexpress, internet-explorer: add page (#22070)
tldr-pages/tldr: pages/windows/iesetup.md

# iesetup

> Build customized Internet Explorer Wizard (`iewzd`) installer.
> Part of Internet Explorer Administration Kit (IEAK).
> Note: You may need to replace the `iesetup` command with your desired IEAK version and executable, such as `Ie11setup` for IE 11.
> More information: <https://learn.microsoft.com/previous-versions/windows/internet-explorer/ie-it-pro/internet-explorer-11/ie11-ieak/iexpress-command-line-options>.

- View the documentation for `iewzd`:

> `tldr {{[-p|--platform]}} windows iewzd`

Reviewer: Why is this formatted as a command example instead of using the `See also` block in the header?

Author: Good question.

1. The header notes already suggest that `iesetup` (Internet Explorer Setup) used for custom enterprise installation might be mistaken with `iewzd` (Internet Explorer Wizard). End-customers who only want to use the default IE installer should go with `iewzd`.

2. IEAK allows users to include `iewzd`-specific arguments into `iesetup`. Placing `tldr {{[-p|--platform]}} windows iewzd` as a command example highlights that we have specific examples on how to use `iewzd` arguments for either `iewzd` or `iesetup`.

Reviewer: I still don't quite see why this wouldn't fit better in the See also section, as it feels like a textbook case for it. However, since there are already precedents where tldr is used for navigation within a software suite, I think it's fine and I won't insist on removing the example.

- Build a custom Internet Explorer Wizard executable, specifying the location of `iewzd` executable:

> `iesetup /c:"iewzd"`

Reviewer: ```suggestion
`iesetup /c:"{{path\to\iewzd.exe}}"`
```

- Apply custom `iewzd` program arguments into the custom installer:

> `iesetup /c:"iewzd {{iewzd_args}}"`

Reviewer: ```suggestion
`iesetup /c:"{{path\to\iewzd.exe}} {{iewzd_args}}"`
```

- Run the build process in a [q]uiet mode, assuming the user account invoking this command as an [a]dministrator or regular [u]ser:

> `iesetup /q:{{a|u}} /c:"iewzd {{iewzd_args}}"`

Reviewer: ```suggestion
`iesetup /q:{{a|u}} /c:"{{path\to\iewzd.exe}} {{iewzd_args}}"`
```

> - [a]sk, [n]ever, or alway[s] [r]estart the computer after successful Internet Explorer installation:

Reviewer: ```suggestion
- Specify the [r]estart behavior after a successful installation ([a]sk, [n]ever, or alway[s]):
```

Reviewer: to conform to the imperative mood

> `iesetup /r:{{a|n|s}} /c:"iewzd {{iewzd_args}}"`

Reviewer: ```suggestion
`iesetup /r:{{a|n|s}} /c:"{{path\to\iewzd.exe}} {{iewzd_args}}"`
```
